bin/o80_log_file_stats

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+import sys
+import time
+import signal_handler
+import o80_pam
+from lightargs import BrightArgs,FileExists
+
+
+def _print_stats(file_path):
+
+    first_observation = None
+    first_iteration = None
+    current_iteration = None
+    first_time_stamp = None
+    current_time_stamp = None
+    missed_iterations = 0
+    nb_observations = 0
+
+    # read_file returns a generator upon instances
+    # of Observations serialized in the file
+    for observation in o80_pam.read_file(file_path):
+        iteration = observation.get_iteration()
+        time_stamp = observation.get_time_stamp()
+        # saving info about the first Observations
+        if first_iteration is None:
+            first_iteration = iteration
+            first_time_stamp = time_stamp
+            current_iteration = iteration
+            first_observation = observation
+        # if 2 observation's iteration are separate by more than 1,
+        # it means the logger missed one observation during recording
+        if iteration-current_iteration>1:
+            missed_iterations += 1
```

Comment:
I think if more than one observation is missing in a row, this would only count one of them.